templates/trivy/trivy-sts.yaml

```diff
@@ -109,6 +109,10 @@ spec:
                 secretKeyRef:
                   name: {{ template "harbor.trivy" . }}
                   key: redisURL
+            {{- if .Values.caBundleSecretName }}
+            - name: "SSL_CERT_DIR"
```

Comment:
does it have to be this `SSL_CERT_DIR` in trivy?

Reply:
SSL_CERT_DIR is an env variable for the OpenSSL libraries this overrides the default with the declared vaule in my environment trivy was not picking up and using the CABundle that was being pushed into the container, so this was my work around to fix it not sure if its the best or not but it does work if the cabundlesecretname variable is set.